packages/integration-karma/test/light-dom/slotting/x/shadow/shadow.html

```diff
@@ -0,0 +1 @@
+<template> </template>
```

Comment:
Not sure the intent of adding this test component.

Reply:
Some components in core are adding style elements to the host element. I simulated that in this component. https://github.com/salesforce/lwc/pull/2322/files#diff-83a10af01820f390441f18122a3c1bcf7b2df1122d09b5f7c6fc6c91e76c567cR5